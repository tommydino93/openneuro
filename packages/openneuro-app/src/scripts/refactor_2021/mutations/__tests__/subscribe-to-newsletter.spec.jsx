import React from 'react'
import { shallow } from 'enzyme'
import SubscribeToNewsletter from '../subscribe-to-newsletter.jsx'

describe('SubscribeToNewsletter mutation', () => {
  it('renders with common props', () => {
    const wrapper = shallow(<SubscribeToNewsletter />)
    expect(wrapper).toMatchSnapshot()
  })
})
