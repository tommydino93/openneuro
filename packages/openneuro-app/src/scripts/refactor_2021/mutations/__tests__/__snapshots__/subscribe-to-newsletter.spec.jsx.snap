// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscribeToNewsletter mutation renders with common props 1`] = `
<Mutation
  mutation={
    Object {
      "definitions": Array [
        Object {
          "directives": Array [],
          "kind": "OperationDefinition",
          "name": Object {
            "kind": "Name",
            "value": "subscribeToNewsletter",
          },
          "operation": "mutation",
          "selectionSet": Object {
            "kind": "SelectionSet",
            "selections": Array [
              Object {
                "alias": undefined,
                "arguments": Array [
                  Object {
                    "kind": "Argument",
                    "name": Object {
                      "kind": "Name",
                      "value": "email",
                    },
                    "value": Object {
                      "kind": "Variable",
                      "name": Object {
                        "kind": "Name",
                        "value": "email",
                      },
                    },
                  },
                ],
                "directives": Array [],
                "kind": "Field",
                "name": Object {
                  "kind": "Name",
                  "value": "subscribeToNewsletter",
                },
                "selectionSet": undefined,
              },
            ],
          },
          "variableDefinitions": Array [
            Object {
              "defaultValue": undefined,
              "directives": Array [],
              "kind": "VariableDefinition",
              "type": Object {
                "kind": "NonNullType",
                "type": Object {
                  "kind": "NamedType",
                  "name": Object {
                    "kind": "Name",
                    "value": "String",
                  },
                },
              },
              "variable": Object {
                "kind": "Variable",
                "name": Object {
                  "kind": "Name",
                  "value": "email",
                },
              },
            },
          ],
        },
      ],
      "kind": "Document",
      "loc": Object {
        "end": 98,
        "start": 0,
      },
    }
  }
>
  <Component />
</Mutation>
`;
